src/VisualStudio/Core/Def/ServicesVisualStudio.csproj

```diff
@@ -100,6 +100,7 @@
     <InternalsVisibleToTest Include="Microsoft.VisualStudio.ProjectSystem.Managed.UnitTests" />
     <InternalsVisibleToTest Include="Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests" />
     <!-- </TODO> -->
+    <InternalsVisibleToTest Include="Roslyn.Services.Editor2.UnitTests" />
```

Comment:
What is this needed for?

Reply:
This is needed so the test can create an instance of an `AbstractSnippetExpansionClient` in order to test the ISnippetExpansionSessionIsActiveService implementation that lives in the VS layer.

Reply:
And I suppose we don't have sufficient test helpers in the VS layer to move the test there?

Reply:
Actually--we do have completion tests in the VS layer. I'll move this up and delete the IVT.